doc/building.md

```diff
@@ -55,6 +55,17 @@ cmake ..
 make -j$(nproc)
 ```
 
+_Note:_ on windows is
+```powershell
+cd nyan
+mkdir build
+cd build
+cmake .. -G "Visual Studio 16 2019"
```

Comment:
the generator depends on your ide target, and i wouldn't hardcode a 4 year old version here. isnt't there a better flag?

Reply:
I don't try to remove the flag to see if it still work or not

Reply:
I work without it